CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+Thanks for wanting to contribute to redux-belt! We are very much looking forward to
+your suggestions, bug reports, and pull requests.
+
+## Submitting bug reports
+
+Have a look at our [issue tracker](https://github.com/store2be/redux-belt/issues.
+If you can't find an issue (open or closed) describing your problem
+(or a very similar one) there, please open a issue with the following details:
+
+- Which versions of node, redux, redux-saga and redux-belt are you using?
+- What are you trying to accomplish?
+- What is the full error you are seeing?
+- How can we reproduce this?
+  - Please quote as much of your code as needed to reproduce (best link to a
+    public repository or a [Gist](https://gist.github.com))
+
+Thank you! We'll try to get back to you as soon as possible.
+
+
+## Submitting feature requests
+
+If you can't find an issue (open or closed) describing your idea on our [issue
+tracker](https://github.com/store2be/redux-belt/issues), open an issue.
+It would be great if you could answer the following questions in your description:
+
+- What do you want to do and how do you expect redux-belt to support you with that?
+- How do you think this can be added to redux-belt?
+- What are possible alternatives?
+- Are there any disadvantages?
+
+Thank you! We'll try to get back to you as soon as possible.
+
+
+## Contribute code to redux-belt
+
+### Setting up redux-belt locally
+
+1. Install node.
+2. Clone this repository.
+3. Install the dependencies.
+4. Now, try running the test suite to confirm everything works for you locally
```

Comment:
A bit vague, I'd say we also need to specify they need `yarn` (although it's obvious, but these guidelines are introductory anyway).